experiments/AtmosGCM/heldsuarez.jl

```diff
@@ -97,65 +97,7 @@ function init_heldsuarez!(problem, bl, state, aux, localgeo, t)
     nothing
 end
 
-function held_suarez_forcing!(
-    bl,
-    source,
-    state,
-    diffusive,
-    aux,
-    t::Real,
-    direction,
-)
-    FT = eltype(state)
-
-    # Parameters
-    T_ref = FT(255)
-
-    # Extract the state
-    ρ = state.ρ
-    ρu = state.ρu
-    ρe = state.ρe
-
-    coord = aux.coord
-    e_int = internal_energy(bl.moisture, bl.orientation, state, aux)
-    T = air_temperature(bl.param_set, e_int)
-    _R_d = FT(R_d(bl.param_set))
-    _day = FT(day(bl.param_set))
-    _grav = FT(grav(bl.param_set))
-    _cp_d = FT(cp_d(bl.param_set))
-    _cv_d = FT(cv_d(bl.param_set))
-    _p0 = FT(MSLP(bl.param_set))
-
-    # Held-Suarez parameters
-    k_a = FT(1 / (40 * _day))
-    k_f = FT(1 / _day)
-    k_s = FT(1 / (4 * _day))
-    ΔT_y = FT(60)
-    Δθ_z = FT(10)
-    T_equator = FT(315)
-    T_min = FT(200)
-    σ_b = FT(7 / 10)
-
-    # Held-Suarez forcing
-    φ = latitude(bl.orientation, aux)
-    p = air_pressure(bl.param_set, T, ρ)
-
-    #TODO: replace _p0 with dynamic surfce pressure in Δσ calculations to account
-    #for topography, but leave unchanged for calculations of σ involved in T_equil
-    σ = p / _p0
-    exner_p = σ^(_R_d / _cp_d)
-    Δσ = (σ - σ_b) / (1 - σ_b)
-    height_factor = max(0, Δσ)
-    T_equil = (T_equator - ΔT_y * sin(φ)^2 - Δθ_z * log(σ) * cos(φ)^2) * exner_p
-    T_equil = max(T_min, T_equil)
-    k_T = k_a + (k_s - k_a) * height_factor * cos(φ)^4
-    k_v = k_f * height_factor
-
-    # Apply Held-Suarez forcing
-    source.ρu -= k_v * projection_tangential(bl, aux, ρu)
-    source.ρe -= k_T * ρ * _cv_d * (T - T_equil)
-    return nothing
-end
+include(joinpath("GCMDriver", "gcm_sources.jl"))
```

Comment:
I think this was kept separate intentionally, correct @LenkaNovak?

Reply:
All the changes look good to me! Thanks, guys!

Reply:
> I think this was kept separate intentionally, correct @LenkaNovak?

Yes, I think this was kept separate intentionally. I haven't run GCM tests for a while and don't have a strong preference. @jiahe23 any thoughts?

Reply:
I guess move it to the TestCase directory is not an option ...

Reply:
We don't have to remove it if it's helpful to keep separate--defining a separate type is easy.

Reply:
I suggest reverting this one -- Held-Suarez should be moving to `TestCase` as Jia says, so best to keep this independent of the GCMDriver.

Reply:
Ok, I left things as is.